### PR TITLE
workflow type in project settings and enhanced workflow page update

### PR DESCRIPTION
**Pull request by GiovannaMonti**
## Description

This PR updates the Project Settings page documentation by adding the following information:
- The Workload and Runtime section has been updated, with a new Deployment options card from which users can choose the project deployment strategy and pipeline runner.
- The Advanced section allows users to migrate existing projects to the Enhanced Workflow.

There are also some modifications to the Enhanced Project Workflow page regarding operations that can now be performed through the Console and other small adjustments.

## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensitive content has been committed

### CODE REVIEW

File: docs/console/project-configuration/project-settings.md [hunk 1]
```diff
@@ -68,13 +68,44 @@ An imagePullSecret is a Kubernetes entity used to authenticate and authorize acc
...
+
+This card allows users to manage some options regarding the Project deployment.
+
+![runtime settings](./img/settings-deployment-options.png)
+
+- **Deployment strategy**: specifies whether the Project deployment follows a [GitOps-based approach](/development_suite/deploy/gitops-based/index.md) or a [pipeline-based approach](/development_suite/deploy/pipeline-based/index.md).
+
+  :::info
+  The choice of the deployment strategy is only available on Projects with the [Enhanced Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) enabled. Standard Projects only support a pipeline-based deployment strategy.
+  :::
```
Comment (albertotessarotto) [inline docs/console/project-configuration/project-settings.md:81]:
```suggestion
  :::
  
```

File: docs/console/project-configuration/project-settings.md [hunk 2]
```diff
@@ -68,13 +68,44 @@ An imagePullSecret is a Kubernetes entity used to authenticate and authorize acc
...
+    - **Pull mode** implies the Project is connected with a GitOps tool, which "pulls" the updated configuration directly from the Project repository.
+
+    - **Push mode** means that a CI/CD pipeline is run in order to push changes to the target environment.
```
Comment (albertotessarotto) [inline docs/console/project-configuration/project-settings.md:84]:
```suggestion
    - In **Push mode**, updates and changes to applications or configurations are initiated and directly pushed from a CI/CD pipeline to the target environment
```

File: docs/console/project-configuration/project-settings.md [hunk 3]
```diff
@@ -68,13 +68,44 @@ An imagePullSecret is a Kubernetes entity used to authenticate and authorize acc
...
+This card allows users to manage some options regarding the Project deployment.
+
+![runtime settings](./img/settings-deployment-options.png)
+
+- **Deployment strategy**: specifies whether the Project deployment follows a [GitOps-based approach](/development_suite/deploy/gitops-based/index.md) or a [pipeline-based approach](/development_suite/deploy/pipeline-based/index.md).
+
+  :::info
+  The choice of the deployment strategy is only available on Projects with the [Enhanced Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) enabled. Standard Projects only support a pipeline-based deployment strategy.
+  :::
+    - **Pull mode** implies the Project is connected with a GitOps tool, which "pulls" the updated configuration directly from the Project repository.
```
Comment (albertotessarotto) [inline docs/console/project-configuration/project-settings.md:82]:
```suggestion
    - In **Pull mode**, through a GitOps tool connection, project configurations saved on the Git repository are always consistent with the project’s Runtime configurations
```

File: docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md [hunk 4]
```diff
@@ -85,30 +85,31 @@ With the first release of this new Workflow, existing Projects will require a mi
...
+- The Company requires that all new Projects are created with the Enhanced Workflow. This means that during the Project creation process the Enhanced Workflow will be the only available workflow option.
+- The Company lets users choose if they want to adopt the Enhanced Workflow on the new Project or keep using the standard one.
+- The Company does not support the creation of Projects with the Enhanced Workflow: new Projects will keep using the standard one.
 
-The Enhanced Project Workflow can be activated by enabling two different feature toggles. This operation can be performed by your Mia-Platform referent, who will choose the option that most fits your needs. 
+If you want to ensure that all Projects in your Company, old and new ones, adopt the Enhanced Workflow, you can open a service request and ask for the support of a Mia-Platform referent in order to enable the `ENABLE_CONF_GENERATION_ON_DEPLOY` feature toggle at Company level. This feature toggle activates the new versioning system and the generation of Kubernetes configurations at the time of deployment.
```
Comment (albertotessarotto) [inline docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md:93]:
```suggestion
If you want to ensure that all Projects in your Company, both old and new ones, adopt the Enhanced Workflow, you can open a service request and ask for the support of a Mia-Platform referent in order to enable the `ENABLE_CONF_GENERATION_ON_DEPLOY` feature toggle at Company level. This feature toggle activates the new versioning system and the generation of Kubernetes configurations at the time of deployment.
```

File: docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md [hunk 5]
```diff
@@ -85,30 +85,31 @@ With the first release of this new Workflow, existing Projects will require a mi
 
 ## Activating the feature
 
-:::info
-In order to activate the feature on a Project or Company, please open a service request and ask for the support of a Mia-Platform referent.
-:::
+The Enhanced Project Workflow can be activated by the user when [creating a new Project](/console/project-configuration/create-a-project.mdx#creation-process). There are different possible scenarios in the Project creation process:
+- The Company requires that all new Projects are created with the Enhanced Workflow. This means that during the Project creation process the Enhanced Workflow will be the only available workflow option.
+- The Company lets users choose if they want to adopt the Enhanced Workflow on the new Project or keep using the standard one.
+- The Company does not support the creation of Projects with the Enhanced Workflow: new Projects will keep using the standard one.
```
Comment (albertotessarotto) [inline docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md:92]: I would enclose both points below on an info to make them more evident

File: docs/console/project-configuration/project-settings.md (hunk 3 above)
Comment (GiovannaMonti) [reply docs/console/project-configuration/project-settings.md:82]:
```suggestion
    - In **Pull mode**, through a GitOps tool connection, Project configurations saved on the Git repository are always consistent with the Project’s Runtime configurations
```

File: docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md (hunk 5 above)
Comment (GiovannaMonti) [reply docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md:92]: done

File: docs/console/project-configuration/project-settings.md [hunk 6]
```diff
@@ -68,13 +68,45 @@ An imagePullSecret is a Kubernetes entity used to authenticate and authorize acc
...
+  :::
+  
+    - In **Pull mode**, through a GitOps tool connection, Project configurations saved on the Git repository are always consistent with the Project’s Runtime configurations
+
+    - In **Push mode**, updates and changes to applications or configurations are initiated and directly pushed from a CI/CD pipeline to the target environment
+
+- **Pipeline runner**: specifies the command line deployment tool used by the Project. This option is not available for Projects that adopt a "Pull" deployment strategy, where no pipeline runner is needed. The chosen tool can either be [Mia-Platform LaunchPad (MLP)](https://github.com/mia-platform/mlp), or another custom tool. 
+  :::info
+  In order to use `mlp`, it is required to have the [Smart Deploy](/development_suite/deploy/overview.md#smart-deploy) feature enabled.
+  :::
```
Comment (fredmaggiowski) [inline docs/console/project-configuration/project-settings.md:90]: this is not true, it's actually the other way around; using mlp enables the smart deploy feature

I'd remove the info box entirely

```suggestion
```

File: docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md [hunk 7]
```diff
@@ -85,30 +85,33 @@ With the first release of this new Workflow, existing Projects will require a mi
 
 ## Activating the feature
 
-:::info
-In order to activate the feature on a Project or Company, please open a service request and ask for the support of a Mia-Platform referent.
-:::
+The Enhanced Project Workflow can be activated by the user when [creating a new Project](/console/project-configuration/create-a-project.mdx#creation-process). There are different possible scenarios in the Project creation process:
+- The Company requires that all new Projects are created with the Enhanced Workflow. This means that during the Project creation process the Enhanced Workflow will be the only available workflow option.
+- The Company lets users choose if they want to adopt the Enhanced Workflow on the new Project or keep using the standard one.
```
Comment (fredmaggiowski) [inline docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md:90]: this option is not valid anymore @MIA-Deltat1995 I think this relates to your sprint activity

the same applies to the line below, please rework directly in this PR

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]

File: docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md [hunk 8]
```diff
@@ -81,26 +81,20 @@ With this feature Users will be able to not only review the configurations but a
...
-- The Company requires that all new Projects are created with the Enhanced Workflow. This means that during the Project creation process the Enhanced Workflow will be the only available workflow option.
-- The Company lets users choose if they want to adopt the Enhanced Workflow on the new Project or keep using the standard one.
-- The Company does not support the creation of Projects with the Enhanced Workflow: new Projects will keep using the standard one.
+The Enhanced Project Workflow can be activated by the user on new Projects enabling the relative switch in the Project workflow card following the [project creation process](/console/project-configuration/create-a-project.mdx#creation-process).
 
 :::info
-If you want to ensure that all Projects in your Company, both old and new ones, adopt the Enhanced Workflow, you can open a service request and ask for the support of a Mia-Platform referent in order to enable the `ENABLE_CONF_GENERATION_ON_DEPLOY` feature toggle at Company level. This feature toggle activates the new versioning system and the generation of Kubernetes configurations at the time of deployment.
```
Comment (MIA-Deltat1995) [inline docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md]: >This feature toggle activates the new versioning system and the generation of Kubernetes configurations at the time of deployment.

This sentence should not be here. Is it better to move or delete it?

**Review by MIA-Deltat1995 — APPROVED** [review]

**Review by albertotessarotto — APPROVED** [review]

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**